src/components/10-atoms/icon/README.md

```diff
@@ -75,21 +85,7 @@ The default behavior set the size of the svgs to 16x16 px, to use original size
 | ------------- |
 | `size="auto"` |
 
-## Using custom icons
```

Comment:
Why did you remove this part?

Reply:
That information is not relevant for the icon component. Just the info that you have to set the size of the SVG is relevant for my opinion. And i moved that info to another place in the readme